public/javascript/main.js

```diff
@@ -305,6 +305,11 @@ $(function(){
   });
 
   // http://baymard.com/labs/country-selector
+  $('.js-select-to-autocomplete').attr({
```

Comment:
A small thing, but it’s generally good practice to avoid querying the DOM more often than is necessary. So, you might query the DOM once and then cache the result in a variable to be reused again later:

    var $select = $('.js-select-to-autocomplete');
    $select.attr({ … });
    $select.selectToAutoComplete();
    $select.on('change', function(){ … });

Or, since you have a fairly small number of methods to call, you could save the overhead of another variable, and just use jQuery chaining:

    $('.js-select-to-autocomplete')
      .attr({ … })
      .selectToAutocomplete()
      .on('change', function(){ … })